lib/include/openturns/OTtestcode.hxx

```diff
@@ -331,6 +332,23 @@ inline void assert_almost_equal(const Sample & a, const Sample & b, Scalar rtol
   }
 }
 
+inline void assert_almost_equal(const Matrix &a, const Matrix &b, Scalar rtol = 1.0e-5, Scalar atol = 1.0e-8)
```

Comment:
great :)

Reply:
If I understand correctly, this means we will be able to test matrix equality. This will not be available in regular code, though, will it? Only in unit tests?

Reply:
Yes. Now me can check matrix equality. It is only in unit tests.
Do you have a specific use case where you need it in the code?

Reply:
It may be convenient sometimes but I am not sure it is worth putting in the code. In Python, users can always convert matrices to numpy arrays for comparison purposes.

Reply:
Sure. IMHO it is fully convenient *only* for tests (C++/Python).

Reply:
Sure. IMHO it is fully convenient *only* for tests (C++/Python).